clients/cpp-console/docs/RaspberryPi.md

```diff
@@ -46,8 +46,35 @@ There are many ways to do development on a Raspberry pi. It may be useful to tak
 
   ```sh
   cd scripts
-  sh ./buildArm32.sh
+  ./buildArm32Linux.sh
   ```
+  
+## Build directly on Linux arm32 with Microsoft Audio Stack
+
+Microsoft Audio Stack can be used to enhance the experience.
+
+Benefits include:
+* Software Audio Echo Cancellation
+* Beamforming
+* Noise Suppression
+* Automatic Gain Control
+* Dereverberation
```

Comment:
I don't think dereverberation is part of MAS. Or maybe this got added recently? Is this from MAS documentations?

Reply:
I got this list off a slide in the deck Ian sent me so I guess it is?

Reply:
Yes, it is one of the improvements in MAS over NUI Audio.